tests/codeception/functional/Install/Content/FlexiblePageCest.php

```diff
@@ -43,20 +61,23 @@ public function testDuplicateScroll(FunctionalTester $I) {
       'type' => 'hs_basic_page',
     ]);
     $I->amOnPage($node->toUrl('edit-form')->toString());
-    $I->click('List additional actions', '#edit-field-hs-page-components-add-more');
-    $I->click('Add Collection');
+    $I->scrollTo('#edit-field-hs-page-components-add-more-browse');
```

Comment:
Paragraph components are now added through a searchable modal, so most tests will have this change.